haste.py

```diff
@@ -28,8 +28,16 @@ async def handler(event):
 
         msg = await event.get_reply_message()
         if msg.photo:
-            await event.respond('Don\'t send photos with your code or errors. Paste the content in del.dog instead.')
+            sent = await event.respond('Don\'t send photos with your code or errors. Paste the content in del.dog instead.', reply_to=msg.reply_to_msg_id)
+            await asyncio.sleep(10)
+            await sent.delete()
+            return
+        if msg.media:
```

Comment:
`.media` covers both photos and documents. Either find a way to use `.media` instead of `.photo` or replace `.media` with `.document`. The last thing looks way nicer and straight to the point. At least for me